axelor-production/src/main/resources/i18n/messages_fr.csv

```diff
@@ -279,6 +282,8 @@
 "Level","Niveau",,
 "List of Wastes Per Product","Liste de Rebuts par produit",,
 "Low","Faible",,
+"Manuf order merge result","Ordre de fabrication après fusion",,
```

Comment:
Is there a reason to write manuf instead of manufacturing ? Otherwise manuf. with a . would be better.